lib/reporters/mini.js

```diff
@@ -216,7 +216,11 @@ MiniReporter.prototype.finish = function (runStatus) {
 	}
 
 	if (runStatus.failFastEnabled === true) {
-		status += '\n\n  ' + colors.failFast('`--fail-fast` is on. Any number of tests may have been skipped');
+		status += '\n\n  ' + colors.information('`--fail-fast` is on. Any number of tests may have been skipped');
+	}
+
+	if (runStatus.hasExclusive === true && runStatus.remainingCount > 0) {
+		status += '\n\n  ' + colors.information('The .only() modifier is used in some tests.', runStatus.remainingCount, plur('test', runStatus.remainingCount), plur('was', 'were', runStatus.remainingCount), 'not run.');
```

Comment:
@ThomasBem this is very petty, but I just noticed there's a period at the end of the message here (and in the verbose reporter). None of the other messages seem to end in a period. If you've got the time could you do a PR to remove them? 😄